packages/workflow/src/Workflow.ts

```diff
@@ -1402,14 +1402,18 @@ export class Workflow {
 		} else if (nodeType.poll) {
 			if (mode === 'manual') {
 				// In manual mode run the poll function
-				const thisArgs = nodeExecuteFunctions.getExecutePollFunctions(
+				const context = nodeExecuteFunctions.getExecutePollFunctions(
 					this,
 					node,
 					additionalData,
 					mode,
 					'manual',
 				);
-				return { data: await nodeType.poll.call(thisArgs) };
+				const data =
+					nodeType instanceof Node
+						? await nodeType.poll(context)
+						: await nodeType.poll.call(context);
+				return { data };
```

Comment:
What's the reason for this change? Seems to be unrelated to PollContext? Type mismatch?

Reply:
A long time ago I started the [initiative](https://github.com/n8n-io/n8n/pull/6464) to stop abusing `this` as the node execution context in the nodes. I want to pursue that again, so that we can reduce a lot of code duplication in the nodes.

This change doesn't need to be in this PR. I'll remove it for now.